modules/user/user.service.js

```diff
@@ -1,5 +1,6 @@
 const { userRepository } = require("./user.repository");
 const { encodeJWT } = require("./jwt.config");
+const { CustomError } = require("../../errorHandler");
```

Comment:
Can't see any using of this

Reply:
This is used in the userLogin method when the password does not match

`if (!matchLoginPass) {
            throw new CustomError("PASSWORD_IS_INCORRECT");
        }`